packages/vue-vuetify/src/controls/DateTimeControlRenderer.vue

```diff
@@ -197,10 +197,10 @@ import {
   VTab,
   VTabs,
   VTextField,
+  VTimePicker,
```

Comment:
The timer breaks, I can no longer select minutes

![Image](https://github.com/user-attachments/assets/a5e7dbde-4f8e-4e61-ab03-1aa38f5c33d0)

See `#dates` example of Vue Vuetify